config/src/content/jcr_root/apps/core/wcm/config/com.adobe.cq.wcm.core.components.internal.services.cloudviewer.CloudViewerConfigServiceImpl.config

```diff
@@ -0,0 +1,16 @@
+# Copyright 2018 Adobe
```

Comment:
should be 2020

Reply:
Good catch. This should have been removed when switching to context-aware configuration.

Reply:
Removed in a691b53